Assets/_BuildingBlocks/Credentials/Scripts/StoredAuthorization/BearerToken.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using UnityEngine.Networking;
+
+namespace Netherlands3D.Credentials.StoredAuthorization
+{
+    [Serializable]
+    public class BearerToken : InferableSingleKey
```

Comment:
Ik zou deze niet van InferableSingleKey laten extenden omdat er teveel code zit in InferableSingleKey dat gaat over query params.

Ik zou vanaf StoredAuthorization 2 subclasses maken: HeaderBasedAuthorization en QueryStringBasedAuthorization - die subclasses kunnen een methode hebben om een (lijst van) header(s) te krijgen, of een lijst van QueryParameters (vergelijkbaar met wat er nu is, maar nu heeft BearerToken beide).

Omdat Headers en QueryParams op twee verschillende manieren afgehandeld moeten worden, kan je dan beter een gemeenschappelijke en toegespitste interface aanbieden

Reply:
Eens, ik twijfelde hier ook over, en de InferableSingleKey was nog een overblijfsel van de enum